client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDriverUri.java

```diff
@@ -31,11 +32,24 @@ public final class TrinoDriverUri
 {
     private static final String JDBC_URL_PREFIX = "jdbc:";
     private static final String JDBC_URL_START = JDBC_URL_PREFIX + "trino:";
+    private static final String LEGACY_PREPARED_STATEMENTS = "legacyPreparedStatements";
```

Comment:
This is not right.  Please add this to `ConnectionProperties`.

Reply:
Done